osbs/conf.py

```diff
@@ -548,7 +548,7 @@ def get_platform_node_selector(self, platform):
         if platform:
             nodeselector_str = self._get_value("node_selector." + platform, self.conf_section,
                                                "node_selector." + platform)
-            if nodeselector_str:
+            if nodeselector_str and '=' in nodeselector_str:
```

Comment:
I think it would be nice to explicitly compare with `'none'`, just because I think it might be quite easy to mis-type a node selector (e.g. 'amd64' instead of 'cpu=amd64' or whatever).